resources/etspy-gpu.yml

```diff
@@ -0,0 +1,19 @@
+# environment file for the standard (non-GPU) runtime environment
+name: etspy
+channels:
+  - conda-forge
+  - nvidia
+  - defaults
```

Comment:
Are the `nvidia` and `defaults` channels necessary here. In the past, `defaults` and `conda-forge` had binaries incompatibilities and I suspect that this is still the case. The cudatoolkit stack in conda-forge is in very good shape and works well, using conda-forge only would avoid any potential issue.